state_management/javascript/http/order-processor/index.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
+const DAPR_STATE_STORE = 'statestore'
+
+async function main() {
+  for (;;) {
+    const order = {orderId: Math.floor(Math.random() * (1000 - 1) + 1)};
+    // Save state into a state store
+    const state = [{
+      key: "orderId",
+      value: order
+    }];
+    await axios.post(`${DAPR_HOST}:${DAPR_HTTP_PORT}/v1.0/state/${DAPR_STATE_STORE}`, state)
```

Comment:
can we also break the code up a little with new lines? I think it would read cleaner with some space between variable definitions and http requests. Same goes with each request.